HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,736 @@
+                "serial_number": 888,
+                "token_id": "0.0.666"
+            }
+    	    ],
+          "links": {
+              "next": null
+          }
+    		}
+    ```
+    
+    - `limit` - The maximum number of airdrops to return in the response. Defaults to `25` with a max of `100`.
+    - `order` - The direction to sort the items in the response. Can be `asc` or `desc` with a default of `asc`.
+    - `receiver.id` - The receiver account the outstanding airdrop was intended for.
+    - `serialnumber` - The specific serial number associated with airdrop. Supports `eq`, `gt`, `gte`, `lt`, and `lte` operators. Only one occurrence is allowed.
+    - `token.id` - The token ID this airdrop is associated with. Supports `eq`, `gt`, `gte`, `lt`, and `lte` operators. Only one occurrence is allowed.
+- New `/api/v1/accounts/{receiverIdOrEvmAddress}/airdrops/pending`) - List of pending airdrops that `receiverIdOrEvmAddress` has not yet claimed.
+    
+    ```json
+       {
+    			"airdrops": [
+            {
+                "amount": 333,
+                "receiver_id": "0.0.999",
+                "sender_id": "0.0.222",
+    						"serial_number": null,
+                "token_id": "0.0.111"
+            },
+            {
+                "amount": 555,
+                "receiver_id": "0.0.999",
+                "sender_id": "0.0.222",
+    						"serial_number": null,
+                "token_id": "0.0.444"
+            },
+            {
+                "amount": null,
+                "receiver_id": "0.0.999",
+                "sender_id": "0.0.777",
+                "serial_number": 888,
+                "token_id": "0.0.666"
+            }
+    	    ],
+          "links": {
+              "next": null
+          }
+    		}
+    ```
+    
+    - `limit` - The maximum number of airdrops to return in the response. Defaults to `25` with a max of `100`.
+    - `order` - The direction to sort the items in the response. Can be `asc` or `desc` with a default of `asc`.
+    - `sender.id` - The sender account that initiated the pending airdrop.
+    - `serialnumber` - The specific serial number associated with airdrop. Supports `eq`, `gt`, `gte`, `lt`, and `lte` operators. Only one occurrence is allowed.
+    - `token.id` - The token ID this airdrop is associated with. Supports `eq`, `gt`, `gte`, `lt`, and `lte` operators. Only one occurrence is allowed.
+
+### Mirror Node Explorer
+
+As one of the main UIs for users on the network, explorers have the opportunity to surface account properties and provide easy management functionality. Explorers may consider
+
+- supporting the ability to set `maxAutomaticAssociations` via a `CryptoUpdate` across multiple wallets
+- highlighting a users current airdrop account status (unlimited, limited or no airdrops) on the account page
+- exposing pending airdrops on the account page. Explorers could add additional logic to inform potential recipient if the sender can support the pending transfer
```

Comment:
What is meant by `the sender can support the pending transfer`? A check if the account has not expired or cancelled the airdrop?

Reply:
Primarily checking that the sender still has sufficient tokens in their balance to complete the airdrop and sufficient HBAR to pay the sender part of the fees (so a user does not attempt to claim an airdrop that cannot succeed).

Reply:
Updated